netdata-installer.sh

```diff
@@ -296,6 +296,9 @@ while [ -n "${1}" ]; do
         NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//--enable-cloud/} --enable-cloud"
       fi
       ;;
+    "--build-json-c")
```

Comment:
What is this for?

Reply:
For forcing the steps to run for testing purposes. Effectively, it's so that people who already have JSON-C installed on their system can test the logic that builds it and verify that Netdata bundles it correctly.

Reply:
Where does it get set by default (in the non-forcing case)

Reply:
I think this is just for forcing build (for example I use it now to test build despite json-c being present in my system).